src/LandingPage.js

```diff
@@ -1,22 +1,22 @@
 import { Link } from "react-router-dom";
```

Comment:
I would consider this a component. It should also have a proper file extension, being that it returns JSX.